doc/source/tutorial/signal.rst

```diff
@@ -1048,8 +1048,598 @@ giving a factor of :math:`\sim 2` speed increase over the straightforward
 implementation.
 
 
+
+.. currentmodule:: scipy.signal.ShortTimeFFT
+.. math::
+    % LaTeX Macros to make the LaTeX formulas more readable:
+    \newcommand{\IC}{{\mathbb{C}}}  % set of complex numbers
+    \newcommand{\IN}{{\mathbb{N}}}  % set of natural numbers
+    \newcommand{\IR}{{\mathbb{R}}}  % set of real numbers
+    \newcommand{\IZ}{{\mathbb{Z}}}  % set of integers
+    \newcommand{\jj}{{\mathbb{j}}}  % imaginary unit
+    \newcommand{\e}{\operatorname{e}}  % Euler's number
+    \newcommand{\dd}{\operatorname{d}} % infinitesimal operator
+    \newcommand{\conj}[1]{\overline{#1}} % complex conjugate
+    \newcommand{\conjT}[1]{\overline{#1^T}} % transposed complex conjugate
+    \newcommand{\inv}[1]{\left(#1\right)^{\!-1}} % inverse
+    % Since the physics package is not loaded, we define the macros ourselves:
+    \newcommand{\vb}[1]{\mathbf{#1}} % vectors and matrices are bold
+
+.. shorthands to make the source more legible:
+.. |ShortTimeFFT| replace:: :class:`ShortTimeFFT <scipy.signal.ShortTimeFFT>`
+.. |old_stft| replace:: `stft <scipy.signal.stft>`
+.. |old_istft| replace:: `istft <scipy.signal.istft>`
+.. |old_spectrogram| replace:: `spectrogram <scipy.signal.spectrogram>`
+
+.. _tutorial_stft:
+
+Short-Time Fourier Transform
+----------------------------
+This section gives some background information on using the |ShortTimeFFT|
+class: For a complex valued signal :math:`x: \IR \mapsto\IC` the Short-time
```

Comment:
Doesn't `\IR \mapsto\IC` mean it maps real signals to complex spectra?  But `ShortTimeFFT.stft(x...` says `The input signal as real or complex valued array`?